esp/templates/survey/questions/short_answer.html

```diff
@@ -2,7 +2,9 @@
 {% load survey %}
 <table class="fullwidth">
 <tr>
-    <td width="35%"><b>{{ name }}</b></td>
-    <td><input type="text" size="{{ input_length }}" name="question_{{ id }}{% if for_class %}_{% templatetag openvariable %} id {% templatetag closevariable %}{% endif %}"/></td>
+    <td>
+    <div style = "text-align: left; font-weight: bold;"> {{ name }} </div>
+    <div style="text-align: left; padding-left:2em; text-indent:-2em;"><input type="text" size="{{ input_length }}" name="question_{{ id }}{% if for_class %}_{% templatetag openvariable %} id {% templatetag closevariable %}{% endif %}"/></div>
```

Comment:
I think you can specify the `font-size` CSS attribute for the input and it will make the font (and box) bigger. It looks like the font is pretty small on the webapp and the normal survey, so might be fine to hardcode it here (instead of in the separate css files), but also keep in mind that we want the webapp font size to be reasonable for the user's device size (in most cases I used `vh`, you can see examples in webapp.css).